tox.ini

```diff
@@ -0,0 +1,16 @@
+[tox]
+envlist = py26, py27, py33, py34, py35, pypy
+
+[testenv]
+passenv = HYPOTHESIS_STORAGE_DIRECTORY
+
+deps =
+  nose-cov
+  hypothesis
+
+commands =
+  nosetests -v --logging-level=DEBUG --with-cov --cov chardet --cov-config .coveragerc
```

Comment:
I honestly don't know much about `tox`, but it looks like for whatever reason it is not using the override you provided when running `pypy`.  You can also tell because the build takes 19 minutes when you look at the results in Travis.